contributing/code/bc.rst

```diff
@@ -213,29 +213,29 @@ Change name                                         No
 Reduce visibility                                   No
 Move to parent class                                Yes
 Add argument without a default value                No
-Add argument with a default value                   No
+Add argument with a default value                   No [7]_ [8]_
 Remove argument                                     Yes [3]_
-Add default value to an argument                    No
+Add default value to an argument                    No [7]_ [8]_
 Remove default value of an argument                 No
 Add type hint to an argument                        No
```

Comment:
I don't think so, if someone calls the method he expects that everything will work. If you add a type hint then not everything will work anymore. I think this change needs a bc layer.